phoenix-core/src/main/java/org/apache/phoenix/compile/CreateIndexCompiler.java

```diff
@@ -110,6 +111,8 @@ private String getValue(PDataType type) {
             return "ARRAY[" + getValue(PDate.INSTANCE) + "]";
         } else if (type instanceof PArrayDataType) {
             return "ARRAY" + type.getSampleValue().toString();
+        } else if (type instanceof PJson) {
+            return null;
```

Comment:
Can we return a simple JSON doc instead of null here?

Reply:
There was a problem with that - initially i was returning '{"a":1}' but when I was running tests creating the index with the expression, I found that the jsonPath in the expression is applied on the sample json and that ran into issue. Didn't debug it further and rather choose to return null.

Reply:
Done